### PR TITLE
Register nerd.is-a.dev

### PR DESCRIPTION
**Pull request by xkcdnerd**
Register nerd.is-a.dev with CNAME record pointing to nerd.is-a.dev.

### CODE REVIEW

**Comment by xkcdnerd** [comment]
hi

File: domains/nerd.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "xkcdnerd",
+           "email": "evanwang330@gmail.com",
+           "discord": "1153506811522076723"
+        },
+    
+        "record": {
+            "CNAME": "nerd.is-a.dev"
```
Comment (CuteDog5695) [inline domains/nerd.json:9]: you cant have the CNAME record as the domain you want to get

File: domains/c:\.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "xkcdnerd",
+           "email": "evanwang330@gmail.com",
+           "discord": "1153506811522076723"
+        },
+    
+        "record": {
+            "CNAME": "c:\.is-a.dev"
```
Comment (CuteDog5695) [inline domains/c:\.json:9]: you cant have the CNAME record as the domain you want to get

File: domains/c:\.json
Comment (CuteDog5695) [inline domains/c:\.json:1]: invalid file since it has non-numerical stuff, please remove those (either rename this file to something else or delete it)

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/nerd.json (hunk 1 above)
Comment (xkcdnerd) [reply domains/nerd.json]: fixed it

**Review by iostpa — CHANGES_REQUESTED** [review]
you didnt fix shit here, all the CNAMES cant be visited since they dont even exist. please just stick to one domain and put your website

**Comment by 21Z** [comment]
wth are u tryna do
the only valid file is domains/nerd.json

**Comment by xkcdnerd** [comment]
hello?

**Review by Stef-DP — CHANGES_REQUESTED** [review]
you should complete your website or add something to it

**Pull request closed without merging**